hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/sigs/order/SigRequirementsTest.java

```diff
@@ -4701,7 +4701,7 @@ void getsTokenRevokeKycWithExtantFreezable() throws Throwable {
     @Test
     void getsTokenRevokeKycWithMissingToken() throws Throwable {
         // given:
-        setupFor(REVOKE_WITH_MISSING_TOKEN);
+        setupFor(REVOKE_WITH_INVALID_TOKEN);
```

Comment:
@Neeharika-Sompalli @tinker-michaelj is this correct?

Reply:
It is correct, the pre-existing scenario was [just renamed to `REVOKE_WITH_INVALID_TOKEN` ](https://github.com/hashgraph/hedera-services/pull/4652/files#diff-f7cdb7b35aa905568d46b48305824b08358616dea92527d94dd9dc698c37eaaeR40). It is true, though, that it's kind of misleading given the name of the test... Maybe renaming it would make stuff clearer 🤔

Reply:
Renaming the test would definitely be helpful 👍

Reply:
Done!